Current Version 2.1.0 crashes on up to date ArchLinux

```
$ ./kubenav 
INFO[0000] (version=2.1.0, branch=HEAD, revision=b06e7cdd0b7682669661187964f9a78ec6d090c2) 
INFO[0000] (go=go1.14.4, user=runner, date=2020-06-28@14:59:13) 
INFO[0000] Version Check: Current 2.1.0, Available 2.1.0 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x11fa79c]

goroutine 1 [running]:
github.com/kubenav/kubenav/pkg/electron/kube.(*Client).Clusters(0xc000389c80, 0xc0003a8230, 0xc0001b20e0, 0xc0003a8260)
	/Users/runner/work/kubenav/kubenav/pkg/electron/kube/kube.go:90 +0x1bc
main.getMenuOptions(0x1800a00, 0xc000389c80, 0xc0003a8230, 0xc0003a8200, 0x0, 0x0, 0x0, 0xc000389c80)
	/Users/runner/work/kubenav/kubenav/cmd/electron/menu.go:135 +0x6d
main.main()
	/Users/runner/work/kubenav/kubenav/cmd/electron/main.go:135 +0x43a

```

--- a/pkg/electron/kube/kube.go
+++ b/pkg/electron/kube/kube.go
@@ -82,13 +82,13 @@ func (c *Client) Clusters() (map[string]Cluster, error) {
 	clusters = make(map[string]Cluster)
 
 	for context, details := range raw.Contexts {
-		cluster := raw.Clusters[details.Cluster]
-
-		clusters[context] = Cluster{
-			ID:        context,
-			Name:      context,
-			URL:       cluster.Server,
-			Namespace: details.Namespace,
+		if cluster, ok := raw.Clusters[details.Cluster]; ok && cluster.Server != "" {
+			clusters[context] = Cluster{
+				ID:        context,
+				Name:      context,
+				URL:       cluster.Server,
+				Namespace: details.Namespace,
+			}
 		}
 	}
 

